pkg/yurthub/server/server.go

```diff
@@ -125,6 +126,7 @@ func readyz(certificateMgr certificate.YurtCertificateManager) http.Handler {
 func getPodList(sharedFactory informers.SharedInformerFactory, nodeName string) http.Handler {
 
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		sharedFactory.Core().V1().Pods().Informer().SetTransform(putil.TransformStripManagedFields())
```

Comment:
I think we should configure `Transform` when podInformer is initialized. how about move `SetTransform` to the following:

https://github.com/openyurtio/openyurt/blob/2fad346e9a49e2684d0efbe1d43608557d3ea6fb/cmd/yurthub/app/config/config.go#L303

Reply:
It makes sense. Done.